test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py

```diff
@@ -184,6 +193,11 @@ def test_pytorch_train_dgl_gpu(
         and image_cuda_version == "cu113"
     ):
         pytest.skip("ecs test for DGL gpu fails for pt 1.10")
+    if (
+        Version(image_framework_version) in SpecifierSet(">=2.0")
+        and image_cuda_version == "cu121"
+    ):
+        pytest.skip("DGL not supported on PT2.0.1 for cuda 12.1")
```

Comment:
Please perform this version check in `conftest.py` before this test function actually starts executing, so that we do not unnecessarily create EC2 instances for this test.

Reply:
done